### PR TITLE
Changed load.data to add flexibility for column locations

### PR DESCRIPTION
**Pull request by ljohansson**

### CODE REVIEW

File: R/load_data.R [hunk 1]
```diff
@@ -10,8 +10,18 @@ run.demo <- function() {
   cat("Select snp data","\n")
   snp.file <- file.choose()
   cat("Read snp data","\n")
-  #In the SNP-array file only column 3,4 and 6 are of interest
-  snpm.data <- read.table(snp.file, skip = 9, header = T, sep = "\t")[,c(3,4,6)]
+
+  #In the SNP-array file only columns 'Chr', 'Position and 'B Allele Freq are of interest
+  #First find row with headers (following) row with '[Data]' in column 1.
+  snp.headerlines <- readLines( snp.file, n=20)
```
Comment (connoratrug) [inline R/load_data.R:16]: 20 seems fine , but is it based on some file standard ?  should it look further is no Data tag is found ?

File: R/load_data.R [hunk 2]
```diff
@@ -10,8 +10,18 @@ run.demo <- function() {
...
+  snp.startdata <- which(grepl(pattern = "Data", x= snp.headerlines))
+  #The column headers are in the following line
+  snp.colnamesrow <- snp.startdata + 1
+  chrom <- which(strsplit(snp.headerlines, "\t")[[snp.colnamesrow]]=="Chr")
+  chrompos <- which(strsplit(snp.headerlines, "\t")[[snp.colnamesrow]]=="Position" )
+  BAF <- which(strsplit(snp.headerlines, "\t")[[snp.colnamesrow]]=="B Allele Freq" )
+  snpm.data <- read.table(snp.file, skip = 9, header = T, sep = "\t")[,c(chrom,chrompos,BAF)]
```
Comment (connoratrug) [inline R/load_data.R:23]: skip 9 could use the startdata now i guess

**Review by connoratrug — APPROVED** [review]
Tested with mock data files, work fine. Did not test with actual data

- Suggest to add .DS_Store to .gitignore
- Suggest to add tests for file importing

File: R/load_data.R (hunk 1 above)
Comment (ljohansson) [reply R/load_data.R:16]: This is a good idea. Do you know how we could implement this?

**Review by connoratrug — APPROVED** [review]

**Pull request merged**